api/auth.js

```diff
@@ -21,12 +21,14 @@ const jwt = require("jsonwebtoken");
 const ExtractJwt = require("passport-jwt").ExtractJwt;
 const customErrors = require("./customErrors");
 const models = require("../models");
-
 /*
  * Create a new JWT for a user or device.
  */
-function createEntityJWT(entity, options) {
+function createEntityJWT(entity, options, access) {
   const payload = entity.getJwtDataValues();
+  if (access) {
+    payload.access = access;
```

Comment:
`access` is a decent name. I was struggling to come up with something for this when I was thinking about it earlier.